tests/unit/test_transfers_script.py

```diff
@@ -0,0 +1,66 @@
+import pytest
```

Comment:
try to write tests at least for one validation fnc.
you need to practice tests wtih expected errors.